src/components/Nav.story.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { Box } from './Box';
+import { Nav, NavItem, NavGroup } from './Nav';
+import { Text } from './Text';
+
+function NavStory() {
+  return (
```

Comment:
Useless bracket function, you can ommit the return then

Reply:
In this particular case there is 2 way to write it this way or with an anonymous function and implicit return but for lot of reasons anonymous functions are not really recommended to write a React component.